addons/website_event_track_live/views/event_track_templates_list.xml

```diff
@@ -1,19 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<template id="tracks_display_list" inherit_id="website_event_track.tracks_display_list">
```

Comment:
I think we need migration scripts to remove the old template from the database. `upgrade_enterprise` is red :red_circle: